### PR TITLE
adding tox file and fixing flake8

### PR DESCRIPTION
**Pull request by morenod**

### CODE REVIEW

File: osde2e-wrapper.py [hunk 1]
```diff
@@ -12,21 +12,21 @@
 #   limitations under the License.
 
 import argparse
-from datetime import datetime
-from elasticsearch_dsl import Search 
+# from datetime import datetime
```
Comment (dry923) [inline osde2e-wrapper.py]: If we don't need these lets just remove them instead of leaving them commented out.

**Comment by dry923** [comment]
@morenod looks good but needs a rebase w/ the last change.

**Comment by morenod** [comment]
@dry923 resolved the rebase

**Comment by morenod** [comment]
@dry923 rebased

**Review by dry923 — APPROVED** [review]
LGTM!

**Pull request merged**